rele/client.py

```diff
@@ -120,7 +120,8 @@ def __init__(self, gc_project_id, credentials, encoder, timeout):
         else:
             self._client = pubsub_v1.PublisherClient(credentials=credentials)
 
-    def publish(self, topic, data, blocking=False, timeout=None, **attrs):
+    def publish(self, topic, data, blocking=False, timeout=None, raise_exception=True,
+                **attrs):
```

Comment:
This is strange indenting. Is this on purpose?

Reply:
I fixed it in [this commit](https://github.com/mercadona/rele/pull/172/commits/f8ad956e42b608f066887be23dd7eed748859231) but for some reason the changes are not showing up in the files changed section. I've also added locally a pre-commit hook with a lint check to make sure this doesn't happen again.